content/en/developing-pixelfed/intro.md

```diff
@@ -9,11 +9,12 @@ weight = 10
 +++
 
 ## Requirements
-- Git
-- PHP
-- Composer
-- Node
-- NPM
```

Comment:
NPM should still be mentioned as it can be packaged separately from Node

Reply:
Good catch, I was under the assumption that all node installs include npm but that may not be a safe asumption to make